### DISCUSSION TITLE
Musicbee overhaul, bugfixes, new features

### DISCUSSION THREAD

**Discussion by jojo2357** [jojo2357/Music-Discord-Rich-Presence#47] [General]
Sorry for such a long break, but here we are, the long awaited 1.6.0 release of MDRP! Let's review whats new:

- New Musicbee integration! MB now requires a [different plugin](https://github.com/jojo2357/MDRP-MusicBee-Bridge) that I made. This will bring a time remaining count **only for musicbee** (see #46 for an explanation)
- Customizable `artist` and `title` tags, perfect if you would like them to be in a different language, or if you think they just get in the way, see the `DiscordPresenceConfig.ini` to change that
- The artist and title length have been increased, so this should reduce getting cut off unnecessarily.
- A lot of behind the scenes changes should reduce resource consumption, and for those that would like to make changes to the code, should now be able to.
- We ditched the groovyrp name, this should have been done earlier, but better late than never.

I fully expect at least 5 new issues in the next week between this and the plugin, so please dont be shy if you are having problems, as there are likely many people with the same problems.

<hr /><em>This discussion was created from the release <a href='https://github.com/jojo2357/Music-Discord-Rich-Presence/releases/tag/1.6.0'>Musicbee overhaul, bugfixes, new features</a>.</em>

**Comment by throwawayjoke644-sudo** [comment]
okay I may be an idiot and this may be over 4 years old now but how do I install this plugin? there doesn't seem to be instructions anywhere

**Reply by jojo2357** [reply]
The [first thing in the readme](https://github.com/jojo2357/Music-Discord-Rich-Presence?tab=readme-ov-file#music-discord-rich-presence)

> Run this by navigating to the [latest release](https://github.com/jojo2357/Music-Discord-Rich-Presence/releases/latest) and downloading the release zip. unzip the contents and use the bat to run.

**Reply by jojo2357** [reply]
Oh shoot you mean the musicbee plugin my bad. 

> There are no special instructions for installation. Add the DLL to the plugins folder, or go through the mb api

https://github.com/jojo2357/MDRP-MusicBee-Bridge/releases/tag/1.1.0
